### ISSUE TITLE
Resource Index Breaking Because of indexQuery

### ISSUE THREAD

**Issue by tysteiman** [ofcold/nova-sortable#7]
After implementing indexQuery thusly:
```
    /**
     * Build an "index" query for the given resource.
     *
     * @param  \Laravel\Nova\Http\Requests\NovaRequest  $request
     * @param  \Illuminate\Database\Eloquent\Builder  $query
     *
     * @return \Illuminate\Database\Eloquent\Builder
     */
    public static function indexQuery(NovaRequest $request, $query)
    {
        $query->when(empty($request->get('orderBy')), function (Builder $q) {
            $q->getQuery()->orders = [];
            return $q->orderBy('sort_order');
        });

        return $query;
    }
```
I'm getting the following error whenever loading my resource's index page:

> Argument 1 passed to App\Nova\FaqCategory::App\Nova\{closure}() must be an instance of App\Nova\Builder, instance of Illuminate\Database\Eloquent\Builder given, called in /var/www/vendor/laravel/framework/src/Illuminate/Database/Concerns/BuildsQueries.php on line 91

Is this an issue with newer versions of Nova, or how can I fix this?

**Comment by tysteiman** [comment]
After changing 

`$query->when(empty($request->get('orderBy')), function (Builder $q) {`

to

`$query->when(empty($request->get('orderBy')), function ($q) {`

It seems to be working. Would be glad to submit a PR to change the README if that's all that needs to be done.

**Comment by tysteiman** [comment]
#8 If you're interested.

**Comment by tysteiman** [comment]
Closed due to #8 being merged.
